types/components/ListGroup.d.ts

```diff
@@ -6,7 +6,7 @@ import { BsPrefixComponent, SelectCallback } from './helpers';
 
 export interface ListGroupProps {
   variant?: 'flush';
-  horizontal?: boolean;
+  horizontal?: boolean | string;
```

Comment:
actually, this should be the union of known breakpoints like in https://github.com/react-bootstrap/react-bootstrap/blob/87ea3552e0e31e550dc896dca3e6ac2967d82ec2/types/components/Navbar.d.ts#L15

Reply:
Good point - lazy on my part - I'll go tweak and push now